packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -491,15 +491,11 @@ export function makeWallet({
     // for the offer.
     const payments = await Promise.all(paymentPs);
 
-    const paymentKeywordRecord = Object.fromEntries(
-      keywords.map((keyword, i) => [keyword, payments[i]]),
+    const paymentKeywordRecord = Data(
+      Object.fromEntries(keywords.map((keyword, i) => [keyword, payments[i]])),
     );
 
-    const seat = E(zoe).offer(
-      inviteP,
-      harden(proposal),
-      harden(paymentKeywordRecord),
-    );
+    const seat = E(zoe).offer(inviteP, harden(proposal), paymentKeywordRecord);
```

Comment:
Why not harden `paymentKeywordRecord` as it was before?

Reply:
No strong reason, just that it seemed redundant with the `Data` applied a few lines earlier (which of course also does `harden`). Also it removes a tiny bit of `prettier`-imposed linecount bloat.